github-for-jira.sd.yml

```diff
@@ -204,6 +204,22 @@ alarms:
     HighSeverityAlarmWhenTooManyELB5xxErrors: null
     LowSeverityAlarmWhenTooManyBackend5xxErrors: null
     LowSeverityAlarmWhenTooManyELB5xxErrors: null
+  eventsQueueSize:
+    MetricName: ApproximateNumberOfMessagesVisible
+    Namespace: AWS/SQS
+    Description: "Lots of messages in the events queue. Follow the runbook: TBD"
```

Comment:
Same here, it is not the events queue

 \+ We need a runbook

Reply:
There's a ticket for this to be addressed separately: ARC-1094